SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -447,10 +466,31 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 
 	/*
 	 * -- EXPERIMENTAL --
-	 * The maximum size of the event tracing file, in bytes
+	 * The maximum size of a event log (non-rotating), synonymous with squid config behavior `event_tracing_single_log_max_file_size_bytes`
+	 */
+	UPROPERTY(Config)
+	int64 EventTracingSingleLogMaxFileSizeBytes;
+
+	/*
+	 * -- EXPERIMENTAL --
+	 * Whether to enable rotating logs, synonymous with squid config behavior `enable_event_tracing_rotating_logs`
+	 */
+	UPROPERTY(Config)
+	bool bEnableEventTracingRotatingLogs;
+
+	/*
+	 * -- EXPERIMENTAL --
+	 * Rotating log file size, synonymous with squid config behavior `event_tracing_rotating_logs_max_file_size_bytes`
+	 */
+	UPROPERTY(Config)
```

Comment:
Nit: Maybe we can "grey out" this setting if bEnableEventTracingRotatingLogs is false.

Reply:
Yeah think we should do that - currently these aren't exposed in the UI so is still a manual ini configuration